package.json

```diff
@@ -90,5 +88,14 @@
       "prettier --write",
       "git add"
     ]
-  }
+  },
+  "pnpm": {
+    "patchedDependencies": {
+      "@vue/test-utils@2.4.6": "patches/@vue__test-utils@2.4.6.patch",
+      "jest-snapshot@29.7.0": "patches/jest-snapshot@29.7.0.patch",
+      "solid-js@1.8.18": "patches/solid-js@1.8.18.patch",
+      "solid-testing-library@0.3.0": "patches/solid-testing-library@0.3.0.patch"
+    }
+  },
+  "packageManager": "pnpm@9.7.1+sha512.faf344af2d6ca65c4c5c8c2224ea77a81a5e8859cbc4e06b1511ddce2f0151512431dd19e6aff31f2c6a8f5f2aced9bd2273e1fed7dd4de1868984059d2c4247"
```

Comment:
What's this?

Reply:
it's a field describing the exact version of the package manager that should be used, tools like `corepack` read this and use that exact version to run the commands etc